nextflow.config

```diff
@@ -0,0 +1,9 @@
+params {
+  inputFilePath = "$baseDir/data/input.fa"
+  fastaSubsetSize = 1
+  outputFileName = "proteinStats.tab"
+  outputDir = "$launchDir/output"
+}
+docker {
```

Comment:
replace this with a call to include the conf/docker.config

Reply:
you'll have to explain this.

Reply:
replace the docker {} bit with:

includeConfig "$baseDir/conf/docker.config"

NOTE: this is also wrong in orffinder.  i'll change that one too